forte_wrapper/hugginface/zero_shot_classifier.py

```diff
@@ -0,0 +1,133 @@
+# Copyright 2021 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Dict, Set
+
+import importlib
+from forte.common import Resources
+from forte.common.configuration import Config
+from forte.data.data_pack import DataPack
+from forte.processors.base import PackProcessor
+from transformers import pipeline
+
+
+class ZeroShotClassifier(PackProcessor):
+    r"""Wrapper of the models on HuggingFace platform with pipeline tag of
+    `zero-shot-classification`.
+    https://huggingface.co/models?pipeline_tag=zero-shot-classification
+    This wrapper could take any model name on HuggingFace platform with pipeline
+    tag of `zero-shot-classification` in configs to make prediction on the user
+    specified entry type in the input pack and the prediction result goes to the
+    user specified attribute name of that entry type in the output pack. User
+    could input the prediction labels in the config with any word or phrase.
+
+    """
+
+    def __init__(self):
+        super().__init__()
+        self.classifier = None
+
+    def set_up(self):
+        device_num = -1
```

Comment:
Maybe we can follow this covention so user can also specify the gpu to use? https://github.com/asyml/forte-wrappers/blob/main/forte_wrapper/allennlp/allennlp_processors.py#L147